### PR TITLE
fix a bug for getBufferParameter

### PR DESCRIPTION
**Pull request by Richard-Yunchao**

### CODE REVIEW

**Comment by Richard-Yunchao** [comment]
Find a bug introduced by my previous patch. Sorry, It is my mistake.

File: sdk/tests/conformance/resources/gl-object-get-calls.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ debug("test getBufferParameter");
 // Test getBufferParameter
 var bufferTypes = [gl.ARRAY_BUFFER, gl.ELEMENT_ARRAY_BUFFER];
 if (contextVersion > 1) {
-  bufferTypes += [gl.COPY_READ_BUFFER, gl.COPY_WRITE_BUFFER, gl.PIXEL_PACK_BUFFER, gl.PIXEL_UNPACK_BUFFER, gl.TRANSFORM_FEEDBACK_BUFFER, gl.UNIFORM_BUFFER];
+  bufferTypes = [gl.ARRAY_BUFFER, gl.ELEMENT_ARRAY_BUFFER, gl.COPY_READ_BUFFER, gl.COPY_WRITE_BUFFER, gl.PIXEL_PACK_BUFFER, gl.PIXEL_UNPACK_BUFFER, gl.TRANSFORM_FEEDBACK_BUFFER, gl.UNIFORM_BUFFER];
```
Comment (zhenyao) [inline sdk/tests/conformance/resources/gl-object-get-calls.js]: You can use concat, say bufferTypes = bufferTypes.concat([...])

**Comment by Richard-Yunchao** [comment]
@zhenyao , You are right. I updated this small change accordingly.

**Comment by kenrussell** [comment]
LGTM

**Pull request merged**